qlib/config.py

```diff
@@ -166,7 +166,7 @@ def update(self, *args, **kwargs):
         "dataset_cache": "DiskDatasetCache",
         "calendar_cache": None,
         # client config
-        "kernels": 16,
+        "kernels": 64,
```

Comment:
You don't have to change the default value in the config.
Please change it whe qlib.init(***)